README.md

```diff
@@ -1,14 +1,81 @@
-# Pixeltable: A Table Interface for ML Data
+<img src="docs/pixeltable-banner.png" width="45%"/>
 
-Pixeltable is a Python library that lets ML engineers and data scientists focus on experiments and models
-without having to deal with the customary data plumbing.
+# Pixeltable: The Multimodal AI Data Plane
+
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+&nbsp;&nbsp;
+![pytest status](https://github.com/pixeltable/pixeltable/actions/workflows/pytest.yml/badge.svg)
+
+Pixeltable is a Python library that lets AI engineers and data scientists focus on
+exploration, modeling, and app development without having to deal with the customary
+data plumbing.
+
+**Pixeltable redefines data infrastructure and workflow orchestration for AI development.**
+It brings together data storage, versioning, and indexing with orchestration and model
+versioning under a declarative table interface, with transformations, model inference,
+and custom logic represented as computed columns.
+
+## Quick Start
+
+If you just want to play around with Pixeltable to see what it's capable of, the easiest way is to run
+the Pixeltable Basics tutorial in colab:
+
+<a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/docs/tutorials/pixeltable-basics.ipynb">
+  <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>
+</a>
+
+## Local Installation
+
+Pixeltable works with Python 3.9, 3.10, or 3.11 running on Linux or MacOS.
+
+```
+pip install pixeltable
+```
+
+To verify that it's working:
+
+```
+import pixeltable as pxt
+cl = pxt.Client()
+```
+
+For more detailed installation instructions, see the
+[Getting Started with Pixeltable](https://pixeltable.github.io/pixeltable/getting-started/)
+guide. Then, check out the
+[Pixeltable Basics](https://pixeltable.github.io/pixeltable/tutorials/pixeltable-basics/)
+tutorial for a tour of its most important features.
+
+## What problems does Pixeltable solve?
+
+Today’s solutions for AI app development require extensive custom coding and infrastructure
+plumbing. Tracking lineage and versions between and across data transformations, models, and
+deployment is cumbersome. Pixeltable is a replacement for traditional data plumbing, providing
+a unified plane for data, models, and orchestration. It removes the data plumbing overhead in
+building and productionizing AI applications.
+
+## Why should you use Pixeltable?
+
+- It gives you transparency and reproducibility
+    - All generated data is automatically recorded and versioned
+    - You will never need to re-run a workload because you lost track of the input data
+- It saves you money
+    - All data changes are automatically incremental
+    - You never need to re-run pipelines from scratch because you’re adding data
+- It provides a zero-effort path to production
```

Comment:
Let's talk about what we want to put in the readme. I think we should stick to things/functionality that is part of the code base as it is today.

Reply:
Pierre and I talked about this for about an hour yesterday. Feel free to suggest specific edits.

Reply:
I removed the bullet point about path to production. Pls suggest any other edits or approve the PR, it's important that we get this shipped.